### ISSUE TITLE
Wrong result if return type is Task<bool>

### ISSUE THREAD

**Issue by ISmellYu** [nenoNaninu/TypedSignalR.Client.DevTools#62]
It appears that if we return `false` from backend it would appear like we errored and `null` will be printed instead of `false`

IChatHub
```C#
[Hub]
public interface IChatHub : IBaseHub
{
	public Task<bool> Test();
	
	public Task<bool> Test2();

	public Task<bool> Test3();

	public Task<bool> Test4();
}
```

ChatHub
```C#
[SignalRHub]
public class ChatHub : Hub<IChatReceiver>, IChatHub
{
	public async Task<bool> Hello()
	{
		throw new NotImplementedException();
	}
	public async Task<bool> Test()
	{
		return true;
	}
	public async Task<bool> Test2()
	{
		return false;
	}
	public Task<bool> Test3()
	{
		return Task.FromResult(true);
	}
	public Task<bool> Test4()
	{
		return Task.FromResult(false);
	}
}
```

![image](https://github.com/nenoNaninu/TypedSignalR.Client.DevTools/assets/38290598/823d333b-a9c4-4d98-9d4f-21ff9dbfd8cc)

Invoke method to change
https://github.com/nenoNaninu/TypedSignalR.Client.DevTools/blob/main/src/signalr-dev-ui/components/HubMethodInvoker.tsx#L31

**Comment by nenoNaninu** [comment]
Thank you for the report! I fixed this issue and released v1.2.4. Please try it!

**Comment by ISmellYu** [comment]
Works now, thanks for resolving the issue.
